test_runner/regress/test_gc_cutoff.py

```diff
@@ -29,15 +24,18 @@ def test_gc_cutoff(neon_env_builder: NeonEnvBuilder, pg_bin: PgBin):
             "image_creation_threshold": "2",
         }
     )
+
+    pageserver_http = env.pageserver.http_client()
+    pageserver_http.configure_failpoints(("after-timeline-gc-removed-layers", "exit"))
+
+    # Use aggressive GC and checkpoint settings, so that we also exercise GC during the test
+    tenant_id = env.initial_tenant
     endpoint = env.endpoints.create_start("main", tenant_id=tenant_id)
     connstr = endpoint.connstr(options="-csynchronous_commit=off")
     pg_bin.run_capture(["pgbench", "-i", "-s10", connstr])
 
-    pageserver_http.configure_failpoints(("after-timeline-gc-removed-layers", "exit"))
-
     for _ in range(5):
         with pytest.raises(subprocess.SubprocessError):
             pg_bin.run_capture(["pgbench", "-P1", "-N", "-c5", "-T500", "-Mprepared", connstr])
         env.pageserver.stop()
-        env.pageserver.start()
-        pageserver_http.configure_failpoints(("after-timeline-gc-removed-layers", "exit"))
+        env.pageserver.start(extra_env_vars={"FAILPOINTS": "after-timeline-gc-removed-layers=exit"})
```

Comment:
This inspired me to do it as in repeated runs there were sometimes completing gc's, because we run gc in the start. I don't think the test expects that (`gc_period` is 10s).

Reply:
Well, these changes are not strictly required, because we might still do a gc which cannot delete anything.